tests/cli/test_cli.py

```diff
@@ -77,5 +96,21 @@ def test_run_cli(inputs: BenchmarkCliTestInput, tmp_path: Path):
 
     assert len(res) == 1
     bench_res = res[0]
-    bench_res.task_results = [tr for tr in bench_res.task_results if tr.task_name != "test-encode-task"]
+    bench_res.task_results = [
+        tr for tr in bench_res.task_results if tr.task_name != "test-encode-task"
+    ]
     assert is_approximately_equal(bench_res.get_main_score(), inputs.score)
+
+
+def test_run_some_models():
```

Comment:
I would move it to a different file called `integration_tests.py` and call the test `test_cli_integration`